packages/xo-remote-parser/src/index.js

```diff
@@ -19,7 +19,14 @@ export const parse = string => {
     object.path = `/${trimStart(rest, '/')}` // the leading slash has been forgotten on client side first implementation
   } else if (type === 'nfs') {
     object.type = 'nfs'
-    const [, host, port, path] = NFS_RE.exec(rest)
+    let host, port, path
+    // some users have a remote with invalid URL which was working before merging the PR https://github.com/vatesfr/xen-orchestra/issues/2299
```

Comment:
This is an issue link, not a PR.

Reply:
Some users have a remote with a colon in the URL, which breaks the parsing since this commit: ...